tests/acceptance.suite.yml

```diff
@@ -36,36 +36,36 @@ env:
           url: 'http://bs-local.com:45691'
           capabilities:
             'browserstack.local': true
-            'name': 'local_test'
+            'name': 'BStack local test'
 
   parallel_0:
     modules:
       config:
         \BrowserStackWebDriver:
           browser: 'chrome'
           capabilities:
-            'name': 'parallel_test'
+            'name': 'BStack parallel test'
 
   parallel_1:
     modules:
       config:
         \BrowserStackWebDriver:
           browser: 'firefox'
           capabilities:
-            'name': 'parallel_test'
+            'name': 'BStack parallel test'
 
   parallel_2:
     modules:
       config:
         \BrowserStackWebDriver:
           browser: 'safari'
           capabilities:
-            'name': 'parallel_test'
+            'name': 'BStack parallel test'
 
   parallel_3:
     modules:
       config:
         \BrowserStackWebDriver:
           browser: 'internet explorer'
           capabilities:
-            'name': 'parallel_test'
+            'name': 'BStack parallel test'
```

Comment:
verify this once.. IE i don't think we have in other repos can be removed (can keep the 3 combinations above chrome ff safari)

Reply:
removed this for now